src/ark/analysis/neighborhood_analysis.py

```diff
@@ -393,8 +393,8 @@ def compute_cell_ratios(neighbors_mat, target_cells, reference_cells, fov_list,
             ref_targ_ratio.append(reference_total / target_total)
 
     # remove nan values for plotting
-    targ_ref_remove_nan = [x for x in targ_ref_ratio if str(x) != 'nan']
-    ref_targ_remove_nan = [x for x in ref_targ_ratio if str(x) != 'nan']
+    targ_ref_remove_nan = [x for x in targ_ref_ratio if x < 15 and str(x) != 'nan']
```

Comment:
```python
targ_ref_ratio = np.array(targ_ref_ratio)
targ_ref_remove_nan= targ_ref_ratio[~np.isnan(targ_ref_ratio)]
```

I'd recommend using casting these two lists to numpy arrays and then using the builtin `np.isnan` function to filter out all `np.nan` values.